tests/changeless.rs

```diff
@@ -14,7 +14,7 @@ fn test_wv(mut rng: impl RngCore) -> impl Iterator<Item = Candidate> {
             value,
             weight: rng.gen_range(0..100),
             input_count: rng.gen_range(1..2),
-            is_segwit: rng.gen_bool(0.5),
+            is_segwit: false,
```

Comment:
Note we never want to choose this randomly in the proptests because we cannot accurately lower bound it.